lib/private/BackgroundJob/JobList.php

```diff
@@ -55,11 +59,10 @@ public function __construct(IDBConnection $connection, IConfig $config, ITimeFac
 		$this->logger = $logger;
 	}
 
-	/**
-	 * @param IJob|class-string<IJob> $job
-	 * @param mixed $argument
-	 */
-	public function add($job, $argument = null): void {
+	public function add($job, $argument = null, int $firstCheck = null): void {
```

Comment:
```suggestion
	public function add($job, $argument = null, ?int $firstCheck = null): void {
```

Or is it okay like that?

Reply:
:potato: :potato: apparently

https://psalm.dev/r/5a7fbe6ae1
https://3v4l.org/K9mhc

Reply:
Roger 🙈

Reply:
Both are equivalent but ?int is a lot clearer